host/xtest/Makefile

```diff
@@ -56,6 +56,12 @@ srcs += xtest_4000.c \
 	adbg/src/adbg_run.c \
 	adbg/src/security_utils_hex.c
 
+ifeq ($(CFG_SECURE_DATA_PATH),y)
+CFLAGS += -DWITH_SECURE_DATA_PATH
```

Comment:
Yes, thanks to https://github.com/OP-TEE/optee_test/blob/master/host/xtest/Makefile#L11.